app/assets/stylesheets/comments.scss

```diff
@@ -0,0 +1,80 @@
+// Place all the styles related to the comments controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+
+.comment-list .row {
+  margin-bottom: 0px;
+}
+.comment-list .panel .panel-heading {
+  padding: 4px 15px;
```

Comment:
Properties should be ordered border, border-top-right-radius, padding, position, top